### ISSUE TITLE
[Chore]: Export types

### ISSUE THREAD

**Issue by SuaYoo** [webrecorder/replayweb.page#376]
### Context

Currently, importing `type Embed` gives `Embed = any`.

### What change would you like to see?

As a developer integrating `<replay-web-page>` into my typescript web app, I'd like `Embed` methods to be typed.

**Comment by SuaYoo** [comment]
@emma-sg I don't think we have a list of events fired by the embed anywhere either, could I tack that onto this task or should I open a separate documentation task?

**Comment by SuaYoo** [comment]
Perhaps we should also rename `Embed` to `ReplaWebPage` to match the tag name?

**Comment by emma-sg** [comment]
Hmm, happy to re-export `Embed` as `ReplaWebPage`, but I can't reproduce getting `any`s as the type of `Embed`
